x-pack/test/api_integration/apis/uptime/rest/filters.ts

```diff
@@ -1,28 +0,0 @@
-/*
- * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
- * or more contributor license agreements. Licensed under the Elastic License
- * 2.0; you may not use this file except in compliance with the Elastic License
- * 2.0.
- */
-
-import { expectFixtureEql } from './helper/expect_fixture_eql';
-import { FtrProviderContext } from '../../../ftr_provider_context';
-
-const getApiPath = (dateRangeStart: string, dateRangeEnd: string, filters?: string) =>
-  `/api/uptime/filters?dateRangeStart=${dateRangeStart}&dateRangeEnd=${dateRangeEnd}${
```

Comment:
I couldn't find any definitions for a route such as `/api/uptime/filters`, therefore, I don't think this test is useful anymore, unless I've missed the route definition or misunderstood the point of this test.